prow.go

```diff
@@ -437,6 +465,20 @@ func (m *jobManager) newJob(job *Job) error {
 			}
 			// CLUSTER_DURATION unused by tests; remove to prevent ci-operator from complaining
 			delete(matchedTarget.MultiStageTestConfiguration.Environment, "CLUSTER_DURATION")
+		}
+		if job.Mode == JobTypeUpgrade {
+			if job.Mode == JobTypeUpgrade {
```

Comment:
Why?

Reply:
That was a small mistake when I was reorganizing some stuff in that section. Removed the double ifs